datarobot_batch_scoring/utils.py

```diff
@@ -252,10 +252,10 @@ def get_config_file():
     """
     home_path = path_join(expanduser('~'), CONFIG_FILENAME)
     cwd_path = path_join(getcwd(), CONFIG_FILENAME)
-    if isfile(home_path):
-        return home_path
-    elif isfile(cwd_path):
+    if isfile(cwd_path):
         return cwd_path
+    elif isfile(home_path):
+        return home_path
```

Comment:
I am very interested how the coverage is calculated for this repo
> Coverage decreased (-0.05%) to 82.863%

I assert that
- there were no changes in lines-of-code of production code;
- there were no changes in the tests of logical paths through this code